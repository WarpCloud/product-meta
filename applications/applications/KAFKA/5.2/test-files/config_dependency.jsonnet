{
  application_type: "KAFKA",
  application_name: "kafka1",
  application_version: "5.2",
  user_config: {},
}
