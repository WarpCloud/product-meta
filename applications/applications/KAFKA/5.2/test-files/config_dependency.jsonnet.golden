{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "kafka1",
            "application_type": "KAFKA"
         },
         "configs": {
            "kafka_cpu_limit": 1,
            "kafka_cpu_request": 0.10000000000000001,
            "kafka_env_list": [
               {
                  "key": "KAFKA_SERVER_PROPERTIES",
                  "value": "log.retention.bytes;;;439804651110|||num.partitions;;;3|||default.replication.factor;;;2"
               }
            ],
            "kafka_memory_limit": 2,
            "kafka_memory_request": 1,
            "kafka_storage_config": {
               "accessModes": [
                  "ReadWriteOnce"
               ],
               "limits": {
                  "blkio.throttle.read_iops_device": 0,
                  "blkio.throttle.write_iops_device": 0
               },
               "size": "512Gi",
               "storageClass": "silver"
            },
            "kafka_tmp_storage": {
               "accessMode": "ReadWriteOnce",
               "limits": {
                  "blkio.throttle.read_iops_device": 0,
                  "blkio.throttle.write_iops_device": 0
               },
               "size": "256Gi",
               "storageClass": "silver"
            }
         },
         "dependencies": [
            {
               "moduleName": "zookeeper",
               "name": "zookeeper"
            }
         ],
         "instance_settings": { },
         "moduleName": "kafka",
         "name": "kafka1-kafka",
         "version": "5.2"
      },
      {
         "annotations": {
            "application_name": "kafka1",
            "application_type": "KAFKA"
         },
         "configs": {
            "kafka_manager_cpu_limit": 1,
            "kafka_manager_cpu_request": 0.10000000000000001,
            "kafka_manager_memory_limit": 2,
            "kafka_manager_memory_request": 1
         },
         "dependencies": [
            {
               "moduleName": "kafka",
               "name": "kafka1-kafka"
            },
            {
               "moduleName": "zookeeper",
               "name": "zookeeper"
            }
         ],
         "instance_settings": { },
         "moduleName": "kafka-manager",
         "name": "kafka1-kafka-manager",
         "version": "1.1"
      }
   ],
   "instance_settings": { },
   "version": ""
}
